charts/pulsar/values.yaml

```diff
@@ -684,6 +684,7 @@ broker:
     minReplicas: 1
     maxReplicas: 3
     metrics: ~
+    behavior: ~
```

Comment:
Out of curiosity, what does the `~` mean? I haven't seen it before. (I see we already use it on the line above.)

Reply:
Good question : `~` resolves to null https://yaml.org/spec/1.2.2/#1032-tag-resolution